smarts/core/chassis.py

```diff
@@ -51,6 +51,14 @@
     RoadSurfacePatch,
 )
 
+vehicle_filepath, controller_parameters = None, None
+with pkg_resources.path(models, "vehicle.urdf") as path:
+    vehicle_filepath = str(path.absolute())
```

Comment:
This `vehicle_filepath` is used for? Couldn't find any place using this variable.

Reply:
Thanks, this is the default value for the `chassis` class for the tests as in many of them instances of chassis are created without explicitly defining the paths, these are default values for them